setup.cfg

```diff
@@ -1,2 +1,3 @@
 [flake8]
-exclude = doc/*
+# Exclude docs and generated files.
+exclude = doc/*,simphony/cuds/meta/,simphony/core/keywords.py,simphony/core/cuba.py
```

Comment:
There is not real reason why the generated python code is not pep8 compatible. This should be fixed.